src/test/java/org/helpberkeley/memberdata/v200/RestaurantTemplateTest.java

```diff
@@ -84,7 +86,7 @@ public void missingEmojiTest() {
         Throwable thrown = catchThrowable(parser::restaurants);
         assertThat(thrown).isInstanceOf(MemberDataException.class);
         assertThat(thrown).hasMessage(
-                "Restaurant Template Error: missing emoji value from column normal, line number 5");
+                "Restaurant Template Error: missing emoji value from column normal, line number 6");
```

Comment:
I know that this isn't code you changed, but as you are touching the codeline, feel free to do cleanup.  Here for example is another place for the String constant / MessageFormat.format pattern.